packages/default-theme/.eslintrc.js

```diff
@@ -16,5 +16,7 @@ module.exports = {
   ],
   plugins: ['prettier'],
   // add your custom rules here
-  rules: {}
+  rules: {
+    'vue/no-parsing-error': [2, { 'x-invalid-end-tag': false }]
```

Comment:
to which line did you need this?
it's not a bad warning to ensure correct end tags; I need more arguments for this :)